website/docs/docs/build/metrics-overview.md

```diff
@@ -76,19 +82,48 @@ metrics:
       {{  Dimension('entity__name') }} > 0 and {{ Dimension(' entity__another_name') }} is not
       null and {{ Metric('metric_name', group_by=['entity_name']) }} > 5
 ```
+</File>
 
 </VersionBlock>
 
-This page explains the different supported metric types you can add to your dbt project.
-
 import SLCourses from '/snippets/_sl-course.md';
 
 <SLCourses/>
 
-### Conversion metrics
+## Default granularity for metircs
```

Comment:
@mirnawong1 This whole section called "Default granularity for metrics" should be version blocked to 1.9+.
Also noting that "metrics" is misspelled in the title (though maybe that's already fixed in production!)